alpine/Dockerfile

```diff
@@ -2,16 +2,20 @@ FROM alpine:3.8
 MAINTAINER VividCortex <containers@vividcortex.com>
 LABEL app=vividcortex
 
-# How to use:
-## docker build --force-rm --build-arg VC_API_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx -t vcimage https://raw.githubusercontent.com/VividCortex/docker/master/alpine/Dockerfile
-#opt1# default hostname : docker run --detach --interactive --tty --name=vividcortex vcimage
-#opt2# override hostname: docker run --env VC_HOSTNAME=mycontainer --detach --interactive --tty --name=vividcortex vcimage
-#opt3# monitor RDS      : docker run --env VC_HOSTNAME=mycontainer --env VC_DRV_MANUAL_HOST_URI=mysql://user:pass@domain.xyz:3306/db --detach --interactive --tty --name=vividcortex vcimage
+# How to build:
+# VC_API_TOKEN can be provided as --build-arg or as --env
+#opt1# build with token: docker build --force-rm --build-arg VC_API_TOKEN=xxxxx -t vcimage https://raw.githubusercontent.com/VividCortex/docker/master/alpine/Dockerfile
```

Comment:
This won't actually work with an `ENV` directive, you have to pass an arg and set the env value from it like this article outlines: https://vsupalov.com/docker-build-pass-environment-variables/

Reply:
ohhhhhhhhhhhhhhh